handlers/incident.go

```diff
@@ -33,6 +35,67 @@ func apiServiceIncidentsHandler(w http.ResponseWriter, r *http.Request) {
 	returnJson(service.Incidents, w, r)
 }
 
+func apiServiceIncidentsHandlerActive(w http.ResponseWriter, r *http.Request) {
+	service, err := findService(r)
+	if err != nil {
+		sendErrorJson(err, w, r)
+		return
+	}
+	var visibleIncidents []incidents.Incident
+	for _, incident := range service.Incidents {
+		if visibilityCheck(incident) == true {
+			incidentVar := *incident
+			reverse(incidentVar.Updates)
+			visibleIncidents = append(visibleIncidents, incidentVar)
+		}
+	}
+	log.Info(fmt.Sprintf("Visible Incidents of Service %v : %v", service.Name, visibleIncidents))
+	returnJson(visibleIncidents, w, r)
+}
+
+func apiSubServiceIncidentsHandlerActive(w http.ResponseWriter, r *http.Request) {
+	service, err := findService(r)
+	if err != nil {
+		sendErrorJson(err, w, r)
+		return
+	}
+	subService, err := findPublicSubService(r, service)
+	if err != nil {
+		sendErrorJson(err, w, r)
+		return
+	}
+
+	var visibleIncidents []incidents.Incident
+	for _, incident := range subService.Incidents {
+		if visibilityCheck(incident) == true {
+			incidentVar := *incident
+			reverse(incidentVar.Updates)
+			visibleIncidents = append(visibleIncidents, incidentVar)
+		}
+	}
```

Comment:
move this to common function and use same function in apiServiceIncidentsHandlerActive and apiSubServiceIncidentsHandlerActive